packages/amplify-provider-awscloudformation/src/utils/delete-from-service.ts

```diff
@@ -8,7 +8,12 @@ import { getSsmSdkParametersDeleteParameters, getSsmSdkParametersGetParametersBy
  * Delete CloudFormation parameters from the service
  */
 export const deleteEnvironmentParametersFromService = async (context: $TSContext, envName: string): Promise<void> => {
-  const appId = resolveAppId(context);
+  let appId;
+  try {
+    appId = resolveAppId(context);
+  } catch (error) {
+    return;
+  }
```

Comment:
nit: If error is ignored, no need to capture it

```suggestion
  } catch {
    return;
  }
```